src/components/ToolTip.svelte

```diff
@@ -0,0 +1,43 @@
+<script context="module">
+  import HelpStrings from "../data/help";
+
+  export const getHelpMessage = (key) => {
+    if (HelpStrings[key]) {
+      return `<span>${HelpStrings[key].text}`;
+    }
+    return "Specific information not found";
+  };
+
+  export const getHelpLink = (key) => {
+    if (HelpStrings[key]) {
+      return HelpStrings[key].link === undefined
+        ? ""
+        : `${HelpStrings[key].link}`;
+    }
+    return "No link found.";
+  };
+</script>
+
+<script>
+  import tippy from "sveltejs-tippy";
+
+  export let props = {
```

Comment:
It's a bit confusing to pass this props stucture into a component. I would construct this dynamically below with `use:tippy`.